include/nccl_ofi_msgbuff.h

```diff
@@ -74,7 +74,9 @@ typedef struct {
 	// Element storage buffer. Allocated in msgbuff_init
 	nccl_ofi_msgbuff_elem_t *buff;
 	// Number of elements in storage buffer
-	uint16_t buff_size;
+	uint16_t max_inprogress;
```

Comment:
I do not get "maximum size of the storage buffer" from `max_inprogress`.  I think this needs a better name.